introduction.md

```diff
@@ -1,3 +1,6 @@
 Hello, this is Sean trying again to introduce myself to the dojo.
 
 My favorite Code School path is JavaScript
+
+* deadly skill Git
+* deadly skill JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?